.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+
+jobs:
+  ci:
+    uses: ./.github/workflows/ci.yml
+  release:
+    name: Release
+    needs: [ci]
+    runs-on: ubuntu-latest
+    environment: production
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)